migration/1629659466000-AddPleromaTL.ts

```diff
@@ -0,0 +1,15 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class addPleromaTL1629659466000 implements MigrationInterface {
```

Comment:
Pleroma reference remaining here (and in filename)

Reply:
Resolved in latest commit